bundles/org.openhab.binding.sony/src/main/java/org/openhab/binding/sony/internal/scalarweb/models/ScalarWebMethod.java

```diff
@@ -0,0 +1,302 @@
+    public static final String GETBROWSERBOOKMARKLIST = "getBrowserBookmarkList";
+    public static final String GETTEXTURL = "getTextUrl";
+    public static final String SETTEXTURL = "setTextUrl";
+
+    // Only Guide service
+    public static final String GETSERVICEPROTOCOLS = "getServiceProtocols";
+    public static final String GETSUPPORTEDAPIINFO = "getSupportedApiInfo";
+
+    // Only System service
+    public static final String GETCURRENTTIME = "getCurrentTime";
+    public static final String GETDEVICEMODE = "getDeviceMode";
+    public static final String GETINTERFACEINFORMATION = "getInterfaceInformation";
+    public static final String GETLEDINDICATORSTATUS = "getLEDIndicatorStatus";
+    public static final String GETNETWORKSETTINGS = "getNetworkSettings";
+    public static final String GETPOSTALCODE = "getPostalCode";
+    public static final String GETPOWERSAVINGMODE = "getPowerSavingMode";
+    public static final String GETPOWERSTATUS = "getPowerStatus";
+    public static final String GETREMOTECONTROLLERINFO = "getRemoteControllerInfo";
+    public static final String GETREMOTEDEVICESETTINGS = "getRemoteDeviceSettings";
+    public static final String GETSTORAGELIST = "getStorageList"; // TODO- figure out
+    public static final String GETSYSTEMINFORMATION = "getSystemInformation";
+    public static final String GETSYSTEMSUPPORTEDFUNCTION = "getSystemSupportedFunction"; // TODO- figure out
+    public static final String GETWOLMODE = "getWolMode";
+    public static final String MOUNTSTORAGE = "mountStorage"; // TODO- figure out
+    public static final String REQUESTREBOOT = "requestReboot";
+    public static final String SETDEVICEMODE = "setDeviceMode";
+    public static final String SETLEDINDICATORSTATUS = "setLEDIndicatorStatus";
+    public static final String SETLANGUAGE = "setLanguage";
+    public static final String SETPOSTALCODE = "setPostalCode";
+    public static final String SETPOWERSAVINGMODE = "setPowerSavingMode";
+    public static final String SETPOWERSTATUS = "setPowerStatus";
+    public static final String SETWOLMODE = "setWolMode";
+    public static final String SWITCHNOTIFICATIONS = "switchNotifications";
+
+    // Only Video Screen service
+    public static final String GETAUDIOSOURCESCREEN = "getAudioSourceScreen";
+    public static final String GETBANNERMODE = "getBannerMode";
+    public static final String GETMULTISCREENMODE = "getMultiScreenMode";
+    public static final String GETPIPSUBSCREENPOSITION = "getPipSubScreenPosition";
+    public static final String GETSCENESETTING = "getSceneSetting";
+    public static final String SETAUDIOSOURCESCREEN = "setAudioSourceScreen";
+    public static final String SETBANNERMODE = "setBannerMode";
+    public static final String SETMULTISCREENMODE = "setMultiScreenMode";
+    public static final String SETPIPSUBSCREENPOSITION = "setPipSubScreenPosition";
+    public static final String SETSCENESETTING = "setSceneSetting";
```

Comment:
Adding underscores to the constants would make it much easier to read.

Reply:
I'd rather leave it as is - it matches the function names this way (makes it easier to remember when you work with this as much as I have) and it get's very verbose if I start adding underscores.